### ISSUE TITLE
CreateTopic/UpdateTopic getting 422 from server (though reporting in logs as 500)

### ISSUE THREAD

**Issue by bglusman** [iggy-rs/iggy-web-ui#3]
When submitting a new topic in Web UI the server logs show
```
2024-01-26T13:42:30.836761Z ERROR server::http::diagnostics: Returning an invalid status code: 422 Unprocessable Entity, IP address: 127.0.0.1:57549, request ID: 01HN303Y5KP8HAZ43A9Z850MTF
```
and the svelte client logs
```
handleErrorStatus: 500 http://127.0.0.1:3000/streams/100/topics
```

**Comment by misrarm** [comment]
Indeed, I cannot create a topic from the IGGY UI.
The creation is done using the CLI.
It seems like the compression is mandatory, and it is not captured in the UI.
From CLI, I used "none"

**Comment by Tchoupinax** [comment]
Hey,
Same issue met here, the compression algorithm is required and not offered in the UI. Could it be added? (Might do a PR if accepted)
Thanks! 🙏

**Comment by Tchoupinax** [comment]
Issue is solved by #6
